docs/guide/idp-app-token.md

```diff
@@ -0,0 +1,69 @@
+# Fake IdP SAML App Canarytoken
+
+## What is an IdP SAML2 App Canarytoken
+
+This Canarytoken works by being installed on your Identity Provider SSO dashboard disguised as a normal app. When the app is opened, the Canarytoken sends an alert with the identity of the user whose dashboard it was used from. It can optionally redirect to a specified URL.
+
+## Creating the Canarytoken
+
+Create a Canarytoken by choosing "IdP App" from the Canarytokens list.
+
+If you want the app to redirect to a specific URL, enter it in the "Redirect URL" box.
+
+Leave a reasonable comment to remind yourself where you will deploy the Canarytoken (e.g. "Fake Salesforce app on Okta").
+
+Tap the Create Canarytoken" button. For a realistic impersonation, select the app you want to impersonate from the dropdown and download an icon to use on your dashboard. You can do this again in future from the token's management page.
+
+In your Identity Provider admin interface create a new SAML2 app, give it the name and icon of your choice, and use the SAML SSO login URL and entity ID provided to configure your token. The app can now be assigned to your users. We recommend training folks about which apps are tokened.
+
+## Installing the Canarytoken
+
+The specific instructions for how to add this token to your suite of SAML applications varies between IdP. Below are detailed instructions for both Microsoft Entra ID (formerly Azure AD) and Okta, the two most popular IdPs.
+
+### Deploying to Azure
+
+As a user in the Azure tenant with at least Enterprise Application Administrator permissions, navigate to the [Enterprise Applications](https://portal.azure.com/#view/Microsoft_AAD_IAM/StartboardApplicationsMenuBlade/~/AppAppsPreview/menuId~/null) panel in the Entra ID Azure Portal. 
+
+![Enterprise Applications](../.vuepress/images/idp_app/entra_enterpriseapp.png)
+
+Select 'New Application', then 'Create a new application' and enter the name of the application as you'd like for it to appear, for example 'Salesforce'. Make sure to select that the application is a 'Non-gallery' application, then click 'Create'.
```

Comment:
Here we're using single quotes as opposed to double quotes earlier in the article. This happens several times below too; pick a standard.